src/Symfony/Component/Mailer/Bridge/Postmark/Transport/PostmarkApiTransport.php

```diff
@@ -51,13 +54,26 @@ public function __toString(): string
 
     protected function doSendApi(SentMessage $sentMessage, Email $email, Envelope $envelope): ResponseInterface
     {
-        $response = $this->client->request('POST', 'https://'.$this->getEndpoint().'/email', [
-            'headers' => [
-                'Accept' => 'application/json',
-                'X-Postmark-Server-Token' => $this->key,
-            ],
-            'json' => $this->getPayload($email, $envelope),
-        ]);
+        try {
+            $response = $this->client->request('POST', 'https://'.$this->getEndpoint().'/email', [
+                'headers' => [
+                    'Accept' => 'application/json',
+                    'X-Postmark-Server-Token' => $this->key,
+                ],
+                'json' => $this->getPayload($email, $envelope),
+            ]);
+        } catch (HttpTransportException $e) {
```

Comment:
This catch won't ever happen, as HttpTransportException is thrown later in this method, in the check `if (200 !== $statusCode) {`